### ISSUE TITLE
Support for nested AND/OR logic inside rule bodies

### ISSUE THREAD

**Issue by tsandall** [open-policy-agent/opa#3183]
One of the most common requests for Rego is to support arbitrary AND/OR logic inside of rule bodies. We should look at different options for how embedded AND/OR logic could be supported without violating safety as well as what impact it would have on policies in the wild.

**Comment by grosser** [comment]
@sspaink is this completed ? ... got a PR or docs link ?

**Comment by sspaink** [comment]
Yes it is 😄  The new `and` and `or` keywords were introduced in the latest release: https://github.com/open-policy-agent/opa/releases/tag/v1.20.0

The work was being tracked in this separate issue: https://github.com/open-policy-agent/opa/issues/7602, sorry didn't know someone was watching this issue!

docs: https://www.openpolicyagent.org/docs/policy-reference/keywords/logical

note that `and` and `or` need to be imported with future keywords:

```rego
import future.keywords.and
import future.keywords.or
```
